terraform/deployments/github/mirror.tf

```diff
@@ -44,7 +44,6 @@ resource "aws_iam_role_policy" "github_action_mirror_repos_policy" {
     Statement = [
       {
         Action = [
-          "codecommit:GitPull",
           "codecommit:GitPush"
```

Comment:
My lack of understanding, but will removing GitPull have any adverse effects. Like it can't then pull (assuming it needs to)?